src/pages/Groups.tsx

```
@@ -85,6 +85,34 @@ export default function Groups() {
       name.toLowerCase().includes(searchLower) ||
       description.toLowerCase().includes(searchLower)
     );
+  }).sort((a, b) => {
+    // Sort by recent activity (most active communities first)
+    const getDTag = (community: any) => community.tags?.find((tag: any) => tag[0] === "d");
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)